roles/openshift_health_checker/library/ocutil.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+"""Interface to OpenShift oc command"""
+
+import os
+import shlex
+import shutil
+import subprocess
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+ADDITIONAL_PATH_LOOKUPS = ['/usr/local/bin', os.path.expanduser('~/bin')]
+
+
+def locate_oc_binary():
```

Comment:
Would including `lib_openshift` be a problem? I expect lots of other stuff in the codebase does and it's just libraries...